creme/polls/templatetags/polls_tags.py

```diff
@@ -60,31 +56,3 @@ def poll_node_number(style, node):
 @register.simple_tag
 def poll_node_css(style, node):
     return style.css(node)
-
-
-@register.simple_tag
-def poll_stats_chart(node):
-    try:
-        if node.type == PollLineType.BOOL:
-            chartpath = 'polls/templatetags/plots/boolean.html'
-            data = [
-                [[percent, 1, f'{answer} − {percent} %']]
-                for answer, _stat, percent in node.answer_stats
-            ]
-        else:
-            chartpath = 'polls/templatetags/plots/number.html'
-            data = [
-                [[percent, str(answer)] for answer, _stat, percent in node.answer_stats],
-            ]
-
-        context = {
-            'node': node,
-            'data': mark_safe(json_encode(data)),  # TODO: use the template-filter 'jsonify' ?
-            'count': len(node.answer_stats),
-        }
-
-        return template.loader.render_to_string(chartpath, context)
-    except Exception:
-        logger.exception('An error occurred in {% poll_stats_chart %}')
-
-        return _('[An error occurred]')
```

Comment:
Comment the code instead + API breaking in changelog (python & related template files)